docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -412,20 +445,21 @@ You can modify the flex item placement through the following properties:
 
     To add space around an item:
 
-    ```
+    ```csharp
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
+    Window.Instance.GetDefaultLayer().Add(flexContainer);
 
     /// Create a flex item
     View item = new View();
 
     /// Add a margin around the item
     item.FlexMargin = new Vector4(10.0f, 10.0f, 10.0f, 10.0f);
-
-    /// Add the item to the container
+    item.Size2D = new Size2D(200, 200);
+    item.BackgroundColor = Color.Red;
     flexContainer.Add(item);
     ```
 
 ## Related Information
 - Dependencies
-  -   Tizen 4.0 and Higher
+  -   Tizen 5.5 and Higher
```

Comment:
Since note about deprecated FlexContainer class is added I changed  dependency to Tizen 5.5 for this reason. It looks like I misunderstood  purpose of this dependency. I will restore it to Tizen 4.0 and Higher